src/GitHub.Logging/ConsoleLogAdapter.cs

```diff
@@ -5,41 +5,40 @@ namespace GitHub.Logging
 {
     public class ConsoleLogAdapter : LogAdapterBase
     {
-        private string GetMessage(string level, string context, string message)
+        private string GetMessage(string context, string message)
         {
             var time = DateTime.Now.ToString("HH:mm:ss.fff tt");
-            var threadId = Thread.CurrentThread.ManagedThreadId;
-            return string.Format("{0} {1} [{2,2}] {3} {4}", time, level, threadId, context, message);
+            return $"{time} {context} {message}";
```

Comment:
None of this was being used. I assumed because it was too noisy.
So I put it back in to be used and dropped some noise.
This is only used in the helper console app.

Reply:
Ah yes. I tweak it every now and then when I'm testing, unit tests can log to the console. 👍